sources/packages/workflow/BPMN/assessment-gateway/assessment-gateway.bpmn

```diff
@@ -1830,6 +1715,124 @@ JSON.stringify(updateProgramInfoJson);</camunda:script>
+        <camunda:out source="numberOfParents" target="numberOfParents" />
+        <camunda:out source="noOfdependants" target="noOfdependants" />
+        <camunda:out source="noOfParentdependants" target="noOfParentdependants" />
+        <camunda:out source="familySize" target="familySize" />
+        <camunda:out source="parentdependantJson" target="parentdependantJson" />
+        <camunda:in source="parent1DependentTable" target="parent1DependentTable" />
+        <camunda:in source="parent2DependentTable" target="parent2DependentTable" />
+        <camunda:out source="totalSecondResidenceLimit" target="totalSecondResidenceLimit" />
+        <camunda:out source="totalLoanPayment" target="totalLoanPayment" />
+        <camunda:out source="totalCost" target="totalCost" />
+        <camunda:out source="netFederalFsc" target="netFederalFsc" />
+        <camunda:out source="netProvincialFsc" target="netProvincialFsc" />
+        <camunda:out source="parentalContribution" target="parentalContribution" />
+        <camunda:out source="applicantSpouseContribution" target="applicantSpouseContribution" />
+        <camunda:out source="netFederalContribution" target="netFederalContribution" />
+        <camunda:out source="netProvincialContribution" target="netProvincialContribution" />
+        <camunda:out source="totalMSOL" target="totalMSOL" />
+        <camunda:out source="otherAllowableCosts" target="otherAllowableCosts" />
+        <camunda:in source="courseLoad" target="courseLoad" />
+        <camunda:out source="dependant11YearsOrUnderExpenses" target="dependant11YearsOrUnderExpenses" />
+        <camunda:out source="dependant12YearsOrOverExpenses" target="dependant12YearsOrOverExpenses" />
+        <camunda:out source="totalDependentExpenses" target="totalDependentExpenses" />
+        <camunda:out source="totalFinancialNeed" target="totalFinancialNeed" />
+        <camunda:out source="totalMiscellaneousCost" target="totalMiscellaneousCost" />
+        <camunda:out source="totalAcademicExpenses" target="totalAcademicExpenses" />
+        <camunda:in source="isProgramOnline" target="isProgramOnline" />
+        <camunda:executionListener event="start">
+          <camunda:script scriptFormat="javascript">var isProgramOnline = false;
+if(execution.getVariable("offeringDelivered") == "online"){
+  isProgramOnline = true;
+}
+execution.setVariable("isProgramOnline", isProgramOnline);</camunda:script>
+        </camunda:executionListener>
+        <camunda:out source="eligibleDependents11YearsOrUnder" target="eligibleDependents11YearsOrUnder" />
+        <camunda:out source="eligibleDependents12YearsOrOver" target="eligibleDependents12YearsOrOver" />
+        <camunda:in source="dependantstatus" target="dependantstatus" />
+        <camunda:in source="partnerStudentStudyWeeks" target="partnerStudentStudyWeeks" />
+        <camunda:in source="totalFundedWeeks" target="weeks" />
```

Comment:
I can see this one mapped as below.

![image](https://user-images.githubusercontent.com/61259237/177864505-27bf69ba-f430-4b17-a40d-f0c0e2a14017.png)

The award should not be mapped in the same way?

![image](https://user-images.githubusercontent.com/61259237/177864657-83801035-19e5-40d6-ad7a-450dae29c22b.png)

Reply:
got it. I thought these `weeks` are different from the `totalFundedWeeks`. that's the reason I gave a different name. nice good catch. will update

Reply:
Good catch @andrewsignori-aot

Reply:
When you will be testing it please check if the `weeks` saved to the assesmment_data JSONB has the expected `totalFundedWeeks` vs `weeks`. I am not sure which one we should display there.